app/Http/Controllers/ProjectController.php

```diff
@@ -16,31 +17,31 @@ class ProjectController extends Controller
      */
     public function index(Request $request)
     {
-        if ($request->get('pid')) {
-            $projects = Project::find(16);
-            $projects->members->pluck('id')->toArray();
+      if ($request->get('pid')) {
+        $projects = Project::find(16);
```

Comment:
Why do you always try to retrieve project with id 16 when a project id is used in the request?

Reply:
Sorry for that, i have create it for test before